data/assets/scene/solarsystem/planets/earth/magnetosphere/magnetosphere.asset

```diff
@@ -25,7 +25,7 @@ local EarthMagnetosphere = {
     MaskingEnabled = true,
     MaskingQuantity = 5,      -- corresponds to "topology"
     MaskingRanges = { { -0, 0 } },  --closed only
-    ColorMethod = "By Quantity",
+    ColorMethod = "ByQuantity",
```

Comment:
Don't know if you're planning to address this based on my previous comment (which seems to be resolved). But again, how about bringing the naming back to how it was before?

Please respond with your thoughts/plans :)

Reply:
Agree that we should try to not break users' asset files on purpose.  And this seems easily preventable (see above comment)

Reply:
Sorry, it looked to me like the other comment was resolved when I recommended on this

PLease resolve this one :) Also, seems like it's fixed by now